metrics/README.md

```diff
@@ -0,0 +1,533 @@
+
```

Comment:
this readme seems to be a copy paste thingy... can you add a proper readme?

Reply:
fixed